Assets/Scripts/Environment/Hangar/BatterySpawnPositionManager.cs

```diff
@@ -9,9 +9,15 @@ public class BatterySpawnPositionManager : MonoBehaviour
     public void Awake()
     {
         // As the game starts, spawn the batteries at their initial position
+
+        if(HangarManager.Batteries.Length > spawnPositions.Length)
+        {
+            Debug.LogError("As we start the game, there aren't enough spawn positions for all batteries. We need to add more.");
+        }
+
         for (int i = 0; i < HangarManager.Batteries.Length; i++)
         {
-            HangarManager.Batteries[i].Container.transform.position = spawnPositions[i].transform.position;
+            HangarManager.Batteries[i].transform.position = spawnPositions[i].transform.position;
```

Comment:
Is there any reason we removed the container? Don't we need it to wrap the two objects as they have separate colliders? (One for physics, one for player/environment interaction)

Reply:
The battery script is now on the container. So when we do: 
HangarManager.Batteries[i].transform.position = spawnPositions[i].transform.position;
it moves the container.